src/JsSortable.php

```diff
@@ -81,15 +81,13 @@ public function init(): void
     public function onReorder($fx = null)
     {
         if (is_callable($fx)) {
-            if ($this->triggered()) {
-                $sortOrders = explode(',', $_POST['order'] ?? '');
-                $source = $_POST['source'] ?? null;
-                $newIdx = $_POST['new_idx'] ?? null;
-                $orgIdx = $_POST['org_idx'] ?? null;
-                $this->set(function () use ($fx, $sortOrders, $source, $newIdx, $orgIdx) {
-                    return call_user_func_array($fx, [$sortOrders, $source, $newIdx, $orgIdx]);
-                });
-            }
+            $sortOrders = explode(',', $_POST['order'] ?? '');
+            $source = $_POST['source'] ?? null;
+            $newIdx = $_POST['new_idx'] ?? null;
+            $orgIdx = $_POST['org_idx'] ?? null;
+            $this->set(function () use ($fx, $sortOrders, $source, $newIdx, $orgIdx) {
+                return call_user_func_array($fx, [$sortOrders, $source, $newIdx, $orgIdx]);
+            });
```

Comment:
To be cleaner here I would propose:
```
    $args = [
            'sortOrders' => explode(',', $_POST['order'] ?? ''),
            'source' => $_POST['source'] ?? null,
            'newIdx' => $_POST['new_idx'] ?? null,
            'orgIdx' => $_POST['org_idx'] ?? null,	
        ];

        $this->set(function () use ($fx, $args) {
            return call_user_func_array($fx, $args);
	});
```
The $args array keys are just for providing extra clarity

Reply:
In will not work, because arguments are unpacked in `call_user_func_array`, but I support this idea to update this:
simply declare vars like before, but in the function, no need to declare and use use :)

Reply:
resolved